.github/workflows/build.yml

```diff
@@ -205,6 +205,7 @@ jobs:
             rm -rf .lake/build/lib/Mathlib
             lake exe cache get || (sleep 1; lake exe cache get)
             lake build --no-build
+            printf '.lake size: %s\n' "$(du -d0 -h .lake)"
```

Comment:
According to the documentation, `du --summarize --human-readable .lake` is equivalent. I think that is slightly more readable; what do you think?

Reply:
I just pushed a commit with the long names, thanks!